src/main/MQ2ImGuiConsole.cpp

```diff
@@ -569,6 +579,21 @@ class ZepMode_ImGuiConsole : public Zep::ZepMode_Standard
 	}
 };
 
+bool TimedMouseHoveredEvent(bool m_isMouseHovered)
+{
+	using namespace std::chrono;
+	if (m_isMouseHovered)
+	{
+		s_mouseHoverStartTimePoint = steady_clock::now();
+		return true;
+	}
+	else
+	{
+		auto m_mouseOverDuration = duration_cast<seconds>(steady_clock::now() - s_mouseHoverStartTimePoint);
+		return (m_mouseOverDuration < s_mouseHoverTimerSecondsDelay) ? true : false;
```

Comment:
using ternary for boolean expression is completely unnecessary. The condition itself is a boolean, so just return it.